### PR TITLE
Update Indonesian translations

### PR DESCRIPTION
**Pull request by srifqi**
This PR updates Indonesian translations by adding missing translations and using better dictions.

-------

**Notes for Indonesian translators:** Please still refer to [the glossary in Weblate](https://hosted.weblate.org/browse/minetest/glossary/id/) while translating Minetest. Any discussions about choice of words/diction etc. can be started in [forum](https://forum.minetest.net/viewforum.php?f=32).

**Catatan untuk penerjemah Indonesia:** Mohon tetap rujuk [glosarium di Weblate](https://hosted.weblate.org/browse/minetest/glossary/id/) ketika menerjemahkan Minetest. Pembahasan tentang pilihan kata/diksi dll. dapat dimulai di [forum](https://forum.minetest.net/viewforum.php?f=32).

### CODE REVIEW

**Review by appgurueu — COMMENTED** [review]
@mnh48 could you give this a look?

**Comment by mnh48** [comment]
> @mnh48 could you give this a look?

@appgurueu  what should I look at?

if you meant to look at the actual translation used word-for-word, I shouldn't disturb Indonesian translation because it's a separate language from Malay where I'm the translator of
like there are vocabulary differences for technical words and even a lot of false-friends for common words

even though we could live our life talking to each other cross-language and tries to pick up the vague meaning from context, we can't just verify the intended meaning in one language using knowledge from the other language alone 😅 

but if you meant to look at the translation to avoid malicious translation (like what happened to Ubuntu before)
I don't see anything malicious in the translations in the PR, all LGTM
plus we all trust srifqi to begin with, he would not be making malicious translations

**Comment by appgurueu** [comment]
> > @mnh48 could you give this a look?
> 
> @appgurueu what should I look at?
> 
> if you meant to look at the actual translation used word-for-word, I shouldn't disturb Indonesian translation because it's a separate language from Malay where I'm the translator of like there are vocabulary differences for technical words and even a lot of false-friends for common words

Ah apologies, my bad, my clueless brain confused something there and associated you with Indonesian... At second thought, I don't think I know any Indonesian speakers in the Minetest community (besides srifqi) who could review this. Do you maybe happen to know any?

I'm not opposed to merging this as-is, but it would be ideal if another Indonesian speaker could give this a look.

**Comment by srifqi** [comment]
The other user who update Indonesian translations of Minetest on GitHub beside me is @SpicyOnta (see PR minetest_game/2758). I am not sure that he is active on GitHub now. Last time I mention him on GitHub was when I updated Indonesian translations of Minetest's builtin back in March 2023 (see PR minetest/13309) and he did not answer/reply.

On Weblate, there is @Linerly who also helps in translating Minetest. Mungkin Kak @Linerly berkenan bantu memeriksa?

File: mods/creative/locale/creative.id.tr [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # textdomain: creative
-Allow player to use creative inventory=Bolehkan pemain memakai inventaris kreatif
-No items to show.=
+Allow player to use creative inventory=Bolehkan pemain menggunakan inventaris kreatif
+No items to show.=Tiada barang untuk ditampilkan.
```
Comment (Linerly) [inline mods/creative/locale/creative.id.tr:3]: I would usually use 'tidak ada' instead of 'tiada', since it sounds more poetic

**Review by Linerly — CHANGES_REQUESTED** [review]
The changes are okay, but I do have a suggestion for one of the strings

File: mods/creative/locale/creative.id.tr (hunk 1 above)
Comment (srifqi) [reply mods/creative/locale/creative.id.tr:3]: Yang mana yang lebih puitis? Kata _tiada_?

Awalnya, _tiada_ dipilih karena keterbatasan ruang. Cuman, kayaknya masih ada cukup ruang di versi yang terakhir.

**Comment by srifqi** [comment]
Thank you for the merge!

... dan terima kasih sudah bantu memeriksa!

**Comment by appgurueu** [comment]
Thank you for the PR, and Linerly for the review!

**Pull request merged**